consensus/core/src/core.rs

```diff
@@ -266,9 +268,9 @@ impl Core {
             .observe(blocks.len() as f64);
 
         // Try to accept them via the block manager
-        let (accepted_blocks, missing_blocks) = self.block_manager.try_accept_blocks(blocks);
+        let (accepted_blocks, missing_block_refs) = self.block_manager.try_accept_blocks(blocks);
 
-        if !accepted_blocks.is_empty() {
+        let accepted_block_refs = if !accepted_blocks.is_empty() {
```

Comment:
I think you can just to in the end:

```
let accepted_block_refs: BTreeSet<BlockRef> = accepted_blocks
                .iter()
                .map(|block| block.reference())
                .collect();
```

and return that. Probably not necessary to if-else here.